Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -165,11 +165,16 @@ public sealed class NetworkIdentity : MonoBehaviour
         /// </summary>
         public bool hasAuthority { get; internal set; }
 
-        /// <summary>
-        /// The set of network connections (players) that can see this object.
-        /// <para>null until OnStartServer was called. this is necessary for SendTo* to work properly in server-only mode.</para>
-        /// </summary>
-        public Dictionary<int, NetworkConnection> observers;
+        // interest management
+        internal HashSet<NetworkConnectionToClient> rebuild = new HashSet<NetworkConnectionToClient>();
```

Comment:
`rebuild` is not an obvious name in the context of NetworkIdentity

Reply:
Do we even really need a "rebuild" on the network identity at all?
If the interest management is handling all the rebuilding anyways, why not have the hashset be temporary/somewhere on the interest management object?